matcher/service/openreview_interface.py

```diff
@@ -68,8 +68,7 @@ def get_all_edges(client, edge_invitation_id, logger=None):
     all_edges = []
     edge_invitation = client.get_invitation(edge_invitation_id)
 
-    edges_grouped_by_paper = openreview.tools.iterget(
-        client.get_grouped_edges,
+    edges_grouped_by_paper = client.get_grouped_edges(
```

Comment:
will there be problems here if more than 1000 objects exist (and only the first 1000 are returned)?

Reply:
there is no pagination in grouped edges, we decided to temporally disabled it.